### ISSUE TITLE
scene.render fails with empty error message becaus installed povray does not work

### ISSUE THREAD

**Issue by Matthias-Schroeter** [conda-forge/vapory-feedstock#4]
### Solution to issue cannot be found in the documentation.

- [x] I checked the documentation.

### Issue

TL:dr the povray executable provided by the package is broken.

- which povray
/home/matthias/anaconda3/envs/vapory/bin/povray
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

- running scene.render("name.png")  in a Jupyter notebook cell results int the following error message:
<class 'bytes'> b''

---------------------------------------------------------------------------
OSError                                   Traceback (most recent call last)
Cell In[10], line 1
----> 1 scene.render("name.png")

File ~/anaconda3/envs/vapory/lib/python3.8/site-packages/vapory/vapory.py:92, in Scene.render(self, outfile, height, width, quality, antialiasing, remove_temp, auto_camera_angle, show_window, tempfile, includedirs, output_alpha)
     89 if auto_camera_angle and width is not None:
     90     self.camera = self.camera.add_args(['right', [1.0*width/height, 0,0]])
---> 92 return render_povstring(str(self), outfile, height, width,
     93                         quality, antialiasing, remove_temp, show_window,
     94                         tempfile, includedirs, output_alpha)

File ~/anaconda3/envs/vapory/lib/python3.8/site-packages/vapory/io.py:131, in render_povstring(string, outfile, height, width, quality, antialiasing, remove_temp, show_window, tempfile, includedirs, output_alpha)
    129 if process.returncode:
    130     print(type(err), err)
--> 131     raise IOError("POVRay rendering failed with the following error: "+err.decode('ascii'))
    133 if return_np_array:
    134     return ppm_to_numpy(buffer=out)

OSError: POVRay rendering failed with the following error: 
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

- rename /home/matthias/anaconda3/envs/vapory/bin/povray to povray_old
-  which povray
/usr/bin/povray

-> everything works 

### Installed packages

```shell
# packages in environment at /home/matthias/anaconda3/envs/vapory:
#
# Name                    Version                   Build  Channel
_libgcc_mutex             0.1                        main  
_openmp_mutex             5.1                       1_gnu  
aiofiles                  22.1.0           py38h06a4308_0  
aiosqlite                 0.18.0           py38h06a4308_0  
anyio                     3.5.0            py38h06a4308_0  
argon2-cffi               21.3.0             pyhd3eb1b0_0  
argon2-cffi-bindings      21.2.0           py38h7f8727e_0  
asttokens                 2.0.5              pyhd3eb1b0_0  
attrs                     23.1.0           py38h06a4308_0  
babel                     2.11.0           py38h06a4308_0  
backcall                  0.2.0              pyhd3eb1b0_0  
beautifulsoup4            4.12.2           py38h06a4308_0  
blas                      1.0                         mkl  
bleach                    4.1.0              pyhd3eb1b0_0  
boost-cpp                 1.70.0               ha2d47e9_1    conda-forge
bottleneck                1.3.5            py38h7deecbd_0  
brotli                    1.0.9                h5eee18b_7  
brotli-bin                1.0.9                h5eee18b_7  
brotli-python             1.0.9            py38h6a678d5_7  
bzip2                     1.0.8                h7f98852_4    conda-forge
ca-certificates           2023.7.22            hbcca054_0    conda-forge
certifi                   2023.7.22          pyhd8ed1ab_0    conda-forge
cffi                      1.15.1           py38h5eee18b_3  
charset-normalizer        2.0.4              pyhd3eb1b0_0  
comm                      0.1.2            py38h06a4308_0  
contourpy                 1.0.5            py38hdb19cb5_0  
cryptography              41.0.3           py38hdda0065_0  
cycler                    0.11.0             pyhd3eb1b0_0  
cyrus-sasl                2.1.28               h52b45da_1  
dbus                      1.13.18              hb2f20db_0  
debugpy                   1.6.7            py38h6a678d5_0  
decorator                 5.1.1              pyhd3eb1b0_0  
defusedxml                0.7.1              pyhd3eb1b0_0  
entrypoints               0.4              py38h06a4308_0  
executing                 0.8.3              pyhd3eb1b0_0  
expat                     2.5.0                h6a678d5_0  
fontconfig                2.14.1               h4c34cd2_2  
fonttools                 4.25.0             pyhd3eb1b0_0  
freetype                  2.12.1               h4a9f257_0  
giflib                    5.2.1                h5eee18b_3  
glib                      2.69.1               he621ea3_2  
gst-plugins-base          1.14.1               h6a678d5_1  
gstreamer                 1.14.1               h5eee18b_1  
icu                       58.2              hf484d3e_1000    conda-forge
idna                      3.4              py38h06a4308_0  
importlib-metadata        6.0.0            py38h06a4308_0  
importlib_metadata        6.0.0                hd3eb1b0_0  
importlib_resources       6.1.0            py38h06a4308_0  
intel-openmp              2023.1.0         hdb19cb5_46305  
ipykernel                 6.25.0           py38h2f386ee_0  
ipython                   8.12.2           py38h06a4308_0  
ipython_genutils          0.2.0              pyhd3eb1b0_1  
ipywidgets                8.0.4            py38h06a4308_0  
jedi                      0.18.1           py38h06a4308_1  
jinja2                    3.1.2            py38h06a4308_0  
jpeg                      9e                   h5eee18b_1  
json5                     0.9.6              pyhd3eb1b0_0  
jsonschema                4.17.3           py38h06a4308_0  
jupyter                   1.0.0            py38h06a4308_8  
jupyter_client            7.4.9            py38h06a4308_0  
jupyter_console           6.6.3            py38h06a4308_0  
jupyter_core              5.3.0            py38h06a4308_0  
jupyter_events            0.6.3            py38h06a4308_0  
jupyter_server            1.23.4           py38h06a4308_0  
jupyter_server_fileid     0.9.0            py38h06a4308_0  
jupyter_server_ydoc       0.8.0            py38h06a4308_1  
jupyter_ydoc              0.2.4            py38h06a4308_0  
jupyterlab                3.6.3            py38h06a4308_0  
jupyterlab_pygments       0.1.2                      py_0  
jupyterlab_server         2.22.0           py38h06a4308_0  
jupyterlab_widgets        3.0.5            py38h06a4308_0  
kiwisolver                1.4.4            py38h6a678d5_0  
krb5                      1.20.1               h143b758_1  
lcms2                     2.12                 h3be6417_0  
ld_impl_linux-64          2.38                 h1181459_1  
lerc                      3.0                  h295c915_0  
libbrotlicommon           1.0.9                h5eee18b_7  
libbrotlidec              1.0.9                h5eee18b_7  
libbrotlienc              1.0.9                h5eee18b_7  
libclang                  14.0.6          default_hc6dbbc7_1  
libclang13                14.0.6          default_he11475f_1  
libcups                   2.4.2                h2d74bed_1  
libdeflate                1.17                 h5eee18b_1  
libedit                   3.1.20221030         h5eee18b_0  
libffi                    3.4.4                h6a678d5_0  
libgcc-ng                 11.2.0               h1234567_1  
libgomp                   11.2.0               h1234567_1  
libllvm14                 14.0.6               hdb19cb5_3  
libpng                    1.6.39               h5eee18b_0  
libpq                     12.15                hdbd6064_1  
libsodium                 1.0.18               h7b6447c_0  
libstdcxx-ng              11.2.0               h1234567_1  
libtiff                   4.2.0                hf544144_3    conda-forge
libuuid                   1.41.5               h5eee18b_0  
libwebp                   1.3.2                h11a3e52_0  
libwebp-base              1.3.2                h5eee18b_0  
libxcb                    1.15                 h7f8727e_0  
libxkbcommon              1.0.1                h5eee18b_1  
libxml2                   2.10.4               hcbfbd50_0  
libxslt                   1.1.37               h2085143_0  
lxml                      4.9.3            py38hdbbb534_0  
lz4-c                     1.9.4                h6a678d5_0  
markupsafe                2.1.1            py38h7f8727e_0  
matplotlib                3.7.2            py38h06a4308_0  
matplotlib-base           3.7.2            py38h1128e8f_0  
matplotlib-inline         0.1.6            py38h06a4308_0  
mistune                   0.8.4           py38h7b6447c_1000  
mkl                       2023.1.0         h213fc3f_46343  
mkl-service               2.4.0            py38h5eee18b_1  
mkl_fft                   1.3.8            py38h5eee18b_0  
mkl_random                1.2.4            py38hdb19cb5_0  
munkres                   1.1.4                      py_0  
mysql                     5.7.24               h721c034_2  
nbclassic                 0.5.5            py38h06a4308_0  
nbclient                  0.5.13           py38h06a4308_0  
nbconvert                 6.5.4            py38h06a4308_0  
nbformat                  5.9.2            py38h06a4308_0  
ncurses                   6.4                  h6a678d5_0  
nest-asyncio              1.5.6            py38h06a4308_0  
notebook                  6.5.4            py38h06a4308_1  
notebook-shim             0.2.2            py38h06a4308_0  
numexpr                   2.8.4            py38hc78ab66_1  
numpy                     1.24.3           py38hf6e8229_1  
numpy-base                1.24.3           py38h060ed82_1  
openjpeg                  2.4.0                h3ad879b_0  
openssl                   3.0.11               h7f8727e_2  
packaging                 23.1             py38h06a4308_0  
pandas                    2.0.3            py38h1128e8f_0  
pandocfilters             1.5.0              pyhd3eb1b0_0  
parso                     0.8.3              pyhd3eb1b0_0  
pcre                      8.45                 h295c915_0  
pexpect                   4.8.0              pyhd3eb1b0_3  
pickleshare               0.7.5           pyhd3eb1b0_1003  
pillow                    10.0.1           py38ha6cbd5a_0  
pip                       23.3             py38h06a4308_0  
pkgutil-resolve-name      1.3.10           py38h06a4308_0  
platformdirs              3.10.0           py38h06a4308_0  
ply                       3.11                     py38_0  
povray                    3.7.0.8              h4bd2361_0    conda-forge
prometheus_client         0.14.1           py38h06a4308_0  
prompt-toolkit            3.0.36           py38h06a4308_0  
prompt_toolkit            3.0.36               hd3eb1b0_0  
psutil                    5.9.0            py38h5eee18b_0  
ptyprocess                0.7.0              pyhd3eb1b0_2  
pure_eval                 0.2.2              pyhd3eb1b0_0  
pycparser                 2.21               pyhd3eb1b0_0  
pygments                  2.15.1           py38h06a4308_1  
pyopenssl                 23.2.0           py38h06a4308_0  
pyparsing                 3.0.9            py38h06a4308_0  
pyqt                      5.15.10          py38h6a678d5_0  
pyqt5-sip                 12.13.0          py38h5eee18b_0  
pyrsistent                0.18.0           py38heee7806_0  
pysocks                   1.7.1            py38h06a4308_0  
python                    3.8.18               h955ad1f_0  
python-dateutil           2.8.2              pyhd3eb1b0_0  
python-fastjsonschema     2.16.2           py38h06a4308_0  
python-json-logger        2.0.7            py38h06a4308_0  
python-tzdata             2023.3             pyhd3eb1b0_0  
pytz                      2023.3.post1     py38h06a4308_0  
pyyaml                    6.0.1            py38h5eee18b_0  
pyzmq                     23.2.0           py38h6a678d5_0  
qt-main                   5.15.2               h7358343_9  
qtconsole                 5.4.2            py38h06a4308_0  
qtpy                      2.2.0            py38h06a4308_0  
readline                  8.2                  h5eee18b_0  
requests                  2.31.0           py38h06a4308_0  
rfc3339-validator         0.1.4            py38h06a4308_0  
rfc3986-validator         0.1.1            py38h06a4308_0  
send2trash                1.8.0              pyhd3eb1b0_1  
setuptools                68.0.0           py38h06a4308_0  
sip                       6.7.12           py38h6a678d5_0  
six                       1.16.0             pyhd3eb1b0_1  
sniffio                   1.2.0            py38h06a4308_1  
soupsieve                 2.5              py38h06a4308_0  
sqlite                    3.41.2               h5eee18b_0  
stack_data                0.2.0              pyhd3eb1b0_0  
tbb                       2021.8.0             hdb19cb5_0  
terminado                 0.17.1           py38h06a4308_0  
tinycss2                  1.2.1            py38h06a4308_0  
tk                        8.6.12               h1ccaba5_0  
tomli                     2.0.1            py38h06a4308_0  
tornado                   6.3.3            py38h5eee18b_0  
traitlets                 5.7.1            py38h06a4308_0  
typing-extensions         4.7.1            py38h06a4308_0  
typing_extensions         4.7.1            py38h06a4308_0  
urllib3                   1.26.18          py38h06a4308_0  
vapory                    0.1.2              pyhd8ed1ab_0    conda-forge
wcwidth                   0.2.5              pyhd3eb1b0_0  
webencodings              0.5.1                    py38_1  
websocket-client          0.58.0           py38h06a4308_4  
wheel                     0.41.2           py38h06a4308_0  
widgetsnbextension        4.0.5            py38h06a4308_0  
xz                        5.4.2                h5eee18b_0  
y-py                      0.5.9            py38h52d8a92_0  
yaml                      0.2.5                h7b6447c_0  
ypy-websocket             0.8.2            py38h06a4308_0  
zeromq                    4.3.4                h2531618_0  
zipp                      3.11.0           py38h06a4308_0  
zlib                      1.2.13               h5eee18b_0  
zstd                      1.5.5                hc292b87_0
```

### Environment info

```shell
active environment : vapory
    active env location : /home/matthias/anaconda3/envs/vapory
            shell level : 2
       user config file : /home/matthias/.condarc
 populated config files : /home/matthias/.condarc
          conda version : 23.9.0
    conda-build version : 3.27.0
         python version : 3.11.5.final.0
       virtual packages : __archspec=1=x86_64
                          __cuda=12.0=0
                          __glibc=2.31=0
                          __linux=5.14.21=0
                          __unix=0=0
       base environment : /home/matthias/anaconda3  (writable)
      conda av data dir : /home/matthias/anaconda3/etc/conda
  conda av metadata url : None
           channel URLs : https://repo.anaconda.com/pkgs/main/linux-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/linux-64
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : /home/matthias/anaconda3/pkgs
                          /home/matthias/.conda/pkgs
       envs directories : /home/matthias/anaconda3/envs
                          /home/matthias/.conda/envs
               platform : linux-64
             user-agent : conda/23.9.0 requests/2.31.0 CPython/3.11.5 Linux/5.14.21-150400.24.33-default opensuse-leap/15.4 glibc/2.31 aau/0.4.3 c/s_Pwgg2yjzSLxqYifP68uQ s/pz6rJeny_jh2nAXwLLFqiw e/mxl-LmdcI-fsJbNX8HteoQ
                UID:GID : 1000:100
             netrc file : None
           offline mode : False
```

**Comment by jan-janssen** [comment]
Can you explain which `povray` version you use locally? `/usr/bin/povray`
